core/tauri-runtime-wry/src/lib.rs

```diff
@@ -2129,34 +2143,48 @@ impl<T: UserEvent> Runtime<T> for Wry<T> {
   }
 
   #[cfg(all(desktop, feature = "system-tray"))]
-  fn system_tray(&self, mut system_tray: SystemTray) -> Result<Self::TrayHandler> {
+  fn system_tray(&self, system_tray: SystemTray) -> Result<Self::TrayHandler> {
```

Comment:
I don't quite remember but I am assuming this function is only called for system tray defined on the `tauri::Builder`, right?

Reply:
yeah, or system trays created in the setup hook...